calendarsubscription/store.go

```diff
@@ -97,11 +98,11 @@ func (b *Store) FindOne(ctx context.Context, id string) (*CalendarSubscription,
 
 	var cs CalendarSubscription
 	err = cs.scanFrom(b.findOne.QueryRowContext(ctx, id).Scan)
-	if err != nil {
-		return nil, err
+	if err == sql.ErrNoRows {
+		return nil, validation.NewFieldError("ID", "Not found")
 	}
 
```

Comment:
I think we should keep the part 
`if err != nil { return nil, err }`
And then
`return &cs, nil`
as the last line.